File: RPN/java/Queue.java
class Queue < E > extends java.util.LinkedList < E > {
    void enQueue(E e) {
        this.add(e);
    }

    E deQueue() {
        return this.poll();
    }

    E front() {
        return this.peek();
    }

    public boolean isEmpty() {
        return this.size() == 0;
    }
}

File: RPN/js/Queue.js
class Queue extends Array {
    enQueue(e) {
        this.push(e);
    }
    deQueue() {
        return this.shift();
    }
    front() {
        return this[0];
    }
    isEmpty() {
        return this.length == 0;
    }
}
exports.Queue = Queue;